charm/charmcraft.yaml

```diff
@@ -40,6 +40,13 @@ config:
         generate the subsequent one-time token authentication.
       type: string
       required: true
+    disallow_fork:
```

Comment:
Should we add the `pull_request` namespace for this?

Reply:
Updated, thank you!